carapace-server/src/main/java/org/carapaceproxy/server/backends/BackendHealthManager.java

```diff
@@ -183,6 +164,12 @@ public void run() {
                 }
             }
             status.setLastProbe(checkResult);
+
+            if (status.isReportedAsUnreachable()) {
+                BACKEND_UPSTATUS_GAUGE.labels(bconf.getHost() + "_" + bconf.getHostPort()).set(0);
```

Comment:
host + hostport?
Isn't it redundant?

Reply:
yes, fixed to bconf.getHost() + "_" + bconf.getPort()